packages/@coorpacademy-components/src/template/common/dashboard/index.js

```diff
@@ -36,6 +39,34 @@ const Dashboard = props => {
         return <Hero hero={hero} welcome={welcome} />;
       case 'battleRequests':
         return <BattleRequestList {...section} />;
+      case 'learningPrioritiesCards': {
+        return (
+          <CardsList
+            {...section}
+            title={
+              <Title
+                {...{
+                  type: 'form-group',
+                  title: section.title,
+                  subtitle: section.subtitle,
+                  icon: {
+                    iconName: 'sign-post',
+                    iconColor: '#A32700',
+                    borderRadius: '12px',
+                    backgroundColor: '#FFDCD1'
+                  },
+                  tag: {
+                    label: `${size(section.cards)}`,
+                    type: 'default',
```

Comment:
(if this could be implicit that would be perfect)

Reply:
ah yes it's possible